src/components/FeaturesDemo/index.tsx

```diff
@@ -0,0 +1,726 @@
+                .current!.querySelectorAll(".diff-bubble")
+                .forEach(el => el.classList.add("animated"));
+
+            await pause(PAUSE_LG, signal);
+        },
+        async function clickOnAccept() {
+            // eslint-disable-next-line @typescript-eslint/non-nullable-type-assertion-style
+            const acceptButton = reportBrowserPageRef
+                .current!.getUnsafePageRef()
+                .current!.querySelector(
+                    'button[data-role-id="accept-button"]',
+                ) as HTMLButtonElement;
+            reportBrowserPageRef.current?.setScrollTop(0);
+            await pause(PAUSE_SM, signal);
+            const acceptButtonSize = acceptButton.getBoundingClientRect();
+
+            const scrollOffset = reportBrowserPageRef.current!.getScrollTop();
+
+            const cursor = reportCursorRef.current!;
+            await cursor.show();
+            await cursor.setPosition(
+                acceptButton.offsetLeft + acceptButtonSize.width / 2 - 6,
+                acceptButton.offsetTop + acceptButtonSize.height / 2 + 40 - 6 - scrollOffset,
+            );
+            void cursor.click();
+            await pause(150, signal);
+            await cursor.hide();
+        },
+        async function acceptScreenshot() {
+            const newTests = _.cloneDeep(
+                (
+                    (useStore.getState().windows[0] as WebBrowserWindowData)
+                        .pageData as ReportPageData
+                ).props.tests,
+            ) as typeof TESTS_SAMPLE_DATA;
+
+            newTests[0].status = Status.Success;
+            newTests[0].children[0].status = Status.Success;
+            newTests[0].children[0].children[0].status = Status.Success;
+            newTests[0].children[0].children[0].attempts.push(SUCCESS_ATTEMPT);
+            newTests[0].children[0].children[0].selectedAttempt = 1;
+
+            store.setWindow(
+                0,
+                {
+                    pageData: { props: { tests: newTests } },
+                },
+                signal,
+            );
+            await pause(PAUSE_LG, signal);
+        },
+    ];
+
+    const runSteps = async () => {
+        try {
+            // eslint-disable-next-line no-constant-condition
+            while (true) {
+                for (const step of steps) {
+                    await step();
+                }
+            }
+        } catch (e: unknown) {
+            // We expect AbortErrors and therefore mute them
+            if (!(e instanceof DOMException) || e.name !== "AbortError") {
+                throw e;
+            }
+        }
+    };
+
+    useEffect(() => {
+        void runSteps();
+
+        return () => {
+            store.reset();
+            controller.abort();
+        };
+    }, []);
+
+    const isHorizontalScreen = useScreenOrientation() === ScreenOrientation.Horizontal;
+
+    const transitions = useTransition(store.windows, {
```

Comment:
docs: https://www.react-spring.dev/docs/components/use-transition

In essence, you:
1. Define transitions using `useTransition` hook
2. Define how each item should be rendered
3. Use `animated.*` components and apply styles from `react-spring` to animate them